content/06.methods.md

```diff
@@ -114,9 +114,9 @@ References can be obtained from the [Broad Genome References on AWS](https://s3.
 
 ##### Assessment of germline variant pathogenicity
 
-For patients with hypermutant samples, we first added population frequency of germline variants using `ANNOVAR` [@doi:10.1093/nar/gkq603] and pathogenicity scoring from ClinVar [@doi:10.1093/nar/gkt1113] using `SnpEff` [@doi:10.4161/fly.19695].
-We then filtered for variants with read depth >= 15, variant allele fraction >= 0.20, and which were observed at < 0.1% allele frequency across each population in the Genome Aggregation Database (see **Key Resources**).
-Finally, we retained variants in genes included in the KEGG MMR gene set (see **Key Resources**), _POLE_, and/or _TP53_ which were ClinVar-annotated as pathogenic (P) or likely pathogenic (L/P) and with review status of >= 2 gold stars.
+For patients with hypermutant samples, we first added population frequency of germline variants using `ANNOVAR` [@doi:10.1093/nar/gkq603] and pathogenicity scoring from ClinVar [@doi:10.1093/nar/gkt1113] using `SnpSift` [@doi:10.3389/fgene.2012.00035].
```

Comment:
So it looks like `SnpSift` called `SnpEff` under the hood? eg, 
https://github.com/d3b-center/OpenPBTA-workflows/blob/1855bc39d0a0b008d99c5d1d8b8b1f0093fb5659/cwl/clinvar_pathogenic_filter.cwl#L87-L92

Reply:
If this change goes through, we will need to update the KR table:
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/93422de6526e87b7f6dc9e3617efd440411314b1/tables/output_tables.Rmd#L501

Reply:
Ahh good point- yes it was a comment from Jung that the part of the alg that does the annotation is SnpSift

Reply:
I opened https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/1611 to track.